src/message/index.js

```diff
@@ -7,4 +7,5 @@ const topicDescriptorProto = protons(require('./topic-descriptor.proto.js'))
 
 exports = module.exports
 exports.rpc = rpcProto
```

Comment:
Do we need this export? I agree that the `rcp.RPC` previously used is not good and exporting the RPC message instead of the module is good.

Reply:
:+1: will remove